README.md

```diff
@@ -353,6 +353,7 @@ Engines, libraries and other helpful things specifically for making games.
 
 * [Allegro][48] - Cross-platform, video game development and multimedia
   library. [``Zlib``][Zlib]
+* [cglm](https://github.com/recp/cglm) - 📽 Optimized OpenGL/Graphics Math (glm) for C. [``MIT``][MIT]
```

Comment:
You still have an inline link. Observe Allegro just above it for how to do it right.